src/js/modules/articles/components/ArticleFeaturedMedia.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import injectSheet from "react-jss";
 
+import CreditLine from "./CreditLine";
```

Comment:
The only lines for credits are `Byline`'s and `Creditline`'s. `Creditline` It's just a term.